packages/mobx-state-tree/src/types/utility-types/late.ts

```diff
@@ -87,6 +89,11 @@ export class Late<C, S, T> extends Type<C, S, T, false> {
         const t = this.getSubType(false)
         return t ? t.isAssignableFrom(type) : false
     }
+
+    getSubTypes() {
+        const subtype = this.getSubType(false)
+        return subtype ? subtype : "cannotDetermine"
```

Comment:
NIT: use a constant? (also on the previous locations)

Reply:
fixed